src/Contact.ts

```diff
@@ -0,0 +1,8 @@
+interface Contact extends Context {
```

Comment:
Are these specific type definitions being created in coordination with the Financial Objects WG?

Reply:
Members of the Context WG is taking part in the Financial Objects WG meetings, with the aim of coordinating, we definitively don't want duplicate standards.

Reply:
At this stage the context data specification is not explicitly saying that a contact or an instrument would be standardised with these interfaces. It is merely an example of how a contact _could_ derive from the base type.

The focus here is primarily on the allowed format for sending data, however I fully expect this group to tackle standardisation of some simple context types at some point, because the financial objects group is mostly concerned with financial objects (although things like contacts/traders and organisations/counterparties is relavant there as well).

There is a disclaimer in the specification markdown that tries to explain that the focus is currently on the shape of context rather than specific types, but maybe a comment in each of the type example files is required as well.

Reply:
When you say 
> I fully expect this group to tackle standardisation of some simple context types at some point

what do you expect that some point to be ? I think standardizing those will be necessary for easier/jump-starting adoption. Not sure if that should be part of this first ratification but intent group did ratify some [standard intents](https://github.com/FDC3/Intents/blob/master/src/standard%20intents.json) as part of their initial ratification if I am not wrong.

Reply:
I totally agree that we need the basic building blocks as soon as possible, ideally in the first round of ratification.

Reply:
Agreed. I was just trying to point out that this specific PR is concerned with defining the format/structure, not the actual types themselves, to limit the scope of what we are discussing. But yes, having standard representations of simple objects like contact and instrument is very important. The examples included here are not supposed to be the definitive versions of contact or instrument, but they could be a good starting point.